eanalyze_staging.sql

```diff
@@ -0,0 +1,1308 @@
+--
+
+CREATE TABLE `home_banner` (
+  `banner_id` int(11) NOT NULL,
+  `image` varchar(255) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+-- --------------------------------------------------------
+
+--
+-- Stand-in structure for view `individual_final_report`
+-- (See below for the actual view)
+--
+CREATE TABLE `individual_final_report` (
+`shipment_id` int(11)
+,`shipment_code` varchar(255)
+,`shipment_date` date
+,`mean_shipment_score` decimal(10,6)
+);
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `instrument`
+--
+
+CREATE TABLE `instrument` (
+  `instrument_id` int(11) NOT NULL,
+  `participant_id` int(11) NOT NULL,
+  `instrument_serial` varchar(45) DEFAULT NULL,
+  `instrument_installed_on` date DEFAULT NULL,
+  `instrument_last_calibrated_on` date DEFAULT NULL,
+  `created_by` varchar(45) DEFAULT NULL,
+  `created_on` datetime DEFAULT NULL,
+  `updated_by` varchar(45) DEFAULT NULL,
+  `updated_on` datetime DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `mail_template`
+--
+
+CREATE TABLE `mail_template` (
+  `mail_temp_id` int(11) NOT NULL,
+  `mail_purpose` varchar(255) NOT NULL,
+  `from_name` varchar(255) DEFAULT NULL,
+  `mail_from` varchar(255) DEFAULT NULL,
+  `mail_cc` varchar(255) DEFAULT NULL,
+  `mail_bcc` varchar(255) DEFAULT NULL,
+  `mail_subject` varchar(255) DEFAULT NULL,
+  `mail_content` text DEFAULT NULL,
+  `mail_footer` text DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `participant`
+--
+
+CREATE TABLE `participant` (
+  `participant_id` int(11) NOT NULL,
+  `unique_identifier` varchar(255) NOT NULL,
+  `lab_name` varchar(500) NOT NULL,
+  `institute_name` varchar(255) DEFAULT NULL,
+  `department_name` varchar(255) DEFAULT NULL,
+  `address` varchar(500) DEFAULT NULL,
+  `city` varchar(255) DEFAULT NULL,
+  `state` varchar(255) DEFAULT NULL,
+  `country` int(11) NOT NULL,
+  `zip` varchar(255) DEFAULT NULL,
+  `long` varchar(255) DEFAULT NULL,
+  `lat` varchar(255) DEFAULT NULL,
+  `shipping_address` varchar(1000) DEFAULT NULL,
+  `funding_source` varchar(255) DEFAULT NULL,
+  `testing_volume` varchar(255) DEFAULT NULL,
+  `enrolled_programs` varchar(255) DEFAULT NULL,
+  `site_type` varchar(255) DEFAULT NULL,
+  `region` varchar(255) DEFAULT NULL,
+  `mobile` varchar(45) DEFAULT NULL,
+  `phone` varchar(45) DEFAULT NULL,
+  `contact_name` varchar(255) DEFAULT NULL,
+  `email` varchar(255) DEFAULT NULL,
+  `affiliation` varchar(45) DEFAULT NULL,
+  `network_tier` int(11) DEFAULT NULL,
+  `created_on` datetime DEFAULT NULL,
+  `created_by` varchar(45) DEFAULT NULL,
+  `updated_on` datetime DEFAULT NULL,
+  `updated_by` varchar(45) DEFAULT NULL,
+  `status` varchar(255) NOT NULL DEFAULT 'inactive'
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
Your script is missing the table primary keys and any existing indexes on the database. Here is the screenshot of MySQL workbench connected to the current ept_staging database.

![image](https://user-images.githubusercontent.com/3584031/106794743-ee408a00-6661-11eb-9bc5-35ab2c36cc80.png)

What are you using to generate this script?

Reply:
I used phpmyadmin to export the script.

Reply:
I use MySQL Workbench and run a dta exprt to generate a full script. It looks like PHP MyAdmin doesn't include existing indexes, keys & stored procedures.

Reply:
okay , let me try with mysql workbench .